backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -4,16 +4,39 @@ import { GovsgMessage } from '@govsg/models'
 import { GovsgVerification } from '@govsg/models/govsg-verification'
 import { WhatsAppApiClient } from '@shared/clients/whatsapp-client.class/types'
 import { sendPasscodeCreationMessage } from '@govsg/services/govsg-verification-service'
+import { Op } from 'sequelize'
+
+const generateSearchOptions = (search: string) => {
```

Comment:
Hmm, I'm currently not keen to take up on this because comments are code smell.

The fact that `generateSearchOptions` was extracted from the middleware into its own function serves as a self-documentation of what's going on.